Src/Sarif.PatternMatcher/AnalyzeCommand.cs

```diff
@@ -73,7 +73,8 @@ public AnalyzeCommand(IFileSystem fileSystem = null)
                     FileVersionInfo fvi = fileSystem.FileVersionInfoGetVersionInfo(Path.Combine(directory, definitions.ValidatorsAssemblyName));
 
                     name = $"{fvi?.CompanyName}/{fvi?.FileDescription}/{name}";
-                    // TBD add version details. Breaks test baselines currently.
+
+                    // TODO: add version details. Breaks test baselines currently. https://dev.azure.com/mseng/1ES/_workitems/edit/2066916
```

Comment:
>https://dev.azure.com/mseng/1ES/_workitems/edit/2066916 [](http://example.com/codeflow?start=83&length=55)

@schlaman-ms Does this align with our coding standard? Shouldn't this reference a GitHub issue? Wanted to cross-check.

Reply:
Is this a new GitHub issue then?
And the work item references that GitHub issue, etc.

Reply:
@schlaman-ms This code base is open source. If someone clicks on the `dev.azure.com` link, is it likely to work? I would understand what the existing standard here is: but at a high level, we should directly reference the GitHub issue here.

Reply:
Well oh my goodness, yes it is open source. Not used to working on open source and well that is obvious now it should be a GitHub issue. Will take care of that and confirm in a new comment.

Reply:
I added a new GitHub issue:
[GitHub issue for copy failure](https://github.com/microsoft/sarif-pattern-matcher/issues/766)

In the ADO work item, I added a link to the newly created GitHub issue.
[ADO work item](https://dev.azure.com/mseng/1ES/_workitems/edit/2071932)